deployment/cdk/opensearch-service-migration/lib/service-stacks/reindex-from-snapshot-stack.ts

```diff
@@ -67,6 +68,7 @@ export class ReindexFromSnapshotStack extends MigrationServiceCore {
         });
         const s3Uri = `s3://migration-artifacts-${this.account}-${props.stage}-${this.region}/rfs-snapshot-repo`;
         let rfsCommand = `/rfs-app/runJavaWithClasspath.sh com.rfs.RfsMigrateDocuments --s3-local-dir /tmp/s3_files --s3-repo-uri ${s3Uri} --s3-region ${this.region} --snapshot-name rfs-snapshot --lucene-dir '/lucene' --target-host ${osClusterEndpoint}`
+        rfsCommand = props.otelCollectorEnabled ? rfsCommand.concat(` --otelCollectorEndpoint http://localhost:${OtelCollectorSidecar.OTEL_CONTAINER_PORT}`) : rfsCommand
```

Comment:
IIRC, this was implemented as `--otel-collector-endpoint` in RFS

Reply:
Good catch - our  tests should have caught this, looks like we aren't running backfill with our python-e2e cases, created jira for this https://opensearch.atlassian.net/browse/MIGRATIONS-1935?atlOrigin=eyJpIjoiOTEzYTM2ZTRiNmU4NGEyNmJmNjNkZGM3MmU1NmI0MTYiLCJwIjoiaiJ9